samples/cpp/image_generation/CMakeLists.txt

```
@@ -107,4 +107,30 @@ set_target_properties(inpainting PROPERTIES
 install(TARGETS inpainting
         RUNTIME DESTINATION samples_bin/
         COMPONENT samples_bin
-        EXCLUDE_FROM_ALL)
\ No newline at end of file
+        EXCLUDE_FROM_ALL)
+
+# benchmark_text2image
+
+include(FetchContent)
+
+if(POLICY CMP0135)
+    cmake_policy(SET CMP0135 NEW)
+endif()
+
+FetchContent_Declare(cxxopts
+    URL https://github.com/jarro2783/cxxopts/archive/refs/tags/v3.1.1.tar.gz
+    URL_HASH SHA256=523175f792eb0ff04f9e653c90746c12655f10cb70f1d5e6d6d9491420298a08)
+FetchContent_MakeAvailable(cxxopts)
+
+add_executable(benchmark_image benchmark_image.cpp load_image.cpp imwrite.cpp)
+target_include_directories(benchmark_image PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" "${CMAKE_BINARY_DIR}")
```

```suggestion
target_include_directories(benchmark_image PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
```
am I right that binary dir is not required?